src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,17 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents the Postal Code portion of an Address
+ * @author thamsy
+ */
+public class PostalCode {
+	private final String value;
```

Comment:
Give more meaningful class variable names. In all the classes the variable name is 'value'